dll/win32/iernonce/iernonce.cpp

```diff
@@ -40,16 +40,40 @@ RunOnceExProcess(_In_ HWND hwnd,
                  _In_ LPCSTR path,
                  _In_ int nShow)
```

Comment:
```suggestion
                 _In_ LPCSTR pszCmdLine,
                 _In_ int nCmdShow)
```
By the way, regarding this, could you test on Windows what happens, when you explicitly call this function without passing through rundll32 (i.e. using `LoadLibrary` to load the dll, then `GetProcAddress` to retrieve a pointer to the function, then calling it), with different values for `pszCmdLine` and `nCmdShow` ?

Reply:
ok! I will test it later